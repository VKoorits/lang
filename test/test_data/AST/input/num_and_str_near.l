while line = stdin.getline():
	print(4"2: ", line);
