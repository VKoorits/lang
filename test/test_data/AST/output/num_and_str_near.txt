ERROR: this tokens can`t be near:
	INT_NUM_TOKEN: 4
	STRING_TOKEN: 2: 
