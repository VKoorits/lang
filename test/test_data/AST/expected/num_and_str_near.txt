ERROR: this tokens can`t be near:
	INT_NUM_TOKEN: 4, info => 0
	STRING_TOKEN: 2: , info => 0
